Keas.Mvc/Views/PersonAdmin/UploadPeople.cshtml

```diff
@@ -15,7 +15,14 @@
                 <form asp-action="UploadPeople" method="post" enctype="multipart/form-data">
                     <div asp-validation-summary="ModelOnly" class="text-danger"></div>
                     <fieldset>
+
                         <div class="form-horizontal">
+                            <div class="form-group">
+                                <div class="row">
+                                <label class="control-label">Update Existing Users Too:</label>
+                                    <input name="UpdateExisting" type="checkbox" class="form-control form-control-lg" value="true"/>
```

Comment:
try setting checked="checked" and see if that binds.  We should figure out why the bool binding doesn't work first